operator/seldon-operator/bundle/certified/1.1.0/seldon-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,527 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "machinelearning.seldon.io/v1",
+          "kind": "SeldonDeployment",
+          "metadata": {
+            "name": "seldon-model"
+          },
+          "spec": {
+            "name": "test-deployment",
+            "predictors": [
+              {
+                "componentSpecs": [
+                  {
+                    "spec": {
+                      "containers": [
+                        {
+                          "image": "registry.connect.redhat.com/seldonio/mock-classifier@sha256:482ee477c344badcaa80e850f4339db41957f9c2396ae24f9e398b67bd5c184e",
```

Comment:
Oh wow does it have to have a hash for tag? Can it not have the one you did above with tag 1.1.0?

Reply:
not inside the example olm not unfortunately

Reply:
Oh wow, fair enough!